<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="text1">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Быстрый поиск:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Search"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableWidget" name="FIO_t"/>
              </item>
              <item>
               <widget class="QPushButton" name="renew_b">
                <property name="text">
                 <string>Обновить таблицу</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QPushButton" name="add_b">
                <property name="text">
                 <string>Добавить клиента</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="del_b">
                <property name="text">
                 <string>Удалить клиента</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: 1px solid black;

</string>
                </property>
                <property name="text">
                 <string>Информация о клиенте</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="styleSheet">
                     <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Статус:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="status"/>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="styleSheet">
                       <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
                      </property>
                      <property name="text">
                       <string>Тариф:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="tariff"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>ФИО:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="FIO">
                    <property name="echoMode">
                     <enum>QLineEdit::Normal</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Тел.:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="phone"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Активен до:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="Date">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="edit">
                <property name="text">
                 <string>Редактировать ФИО/тел.</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="1" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_8">
                      <item>
                       <widget class="QComboBox" name="add_t">
                        <item>
                         <property name="text">
                          <string>1</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>2</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>3</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>4</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>5</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>6</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>7</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>8</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>9</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>10</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>11</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>12</string>
                         </property>
                        </item>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_13">
                      <item>
                       <widget class="QPushButton" name="ok_d_b">
                        <property name="text">
                         <string>Сохранить</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="no_d_b">
                        <property name="text">
                         <string>Отмена</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item row="0" column="0">
                   <widget class="QPushButton" name="date_b">
                    <property name="text">
                     <string>Продлить абонемент</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="1" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_6">
                      <item>
                       <widget class="QComboBox" name="change_t"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_12">
                      <item>
                       <widget class="QPushButton" name="ok_t_b">
                        <property name="text">
                         <string>Сохранить</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="no_t_b">
                        <property name="text">
                         <string>Отмена</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <spacer name="verticalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item row="0" column="0">
                   <widget class="QPushButton" name="change_b">
                    <property name="text">
                     <string>Сменить тариф</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="text2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QTableWidget" name="tariff_t"/>
              </item>
              <item>
               <widget class="QPushButton" name="renew_tariff">
                <property name="text">
                 <string>Обновить таблицу</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QPushButton" name="add_tariff">
                <property name="text">
                 <string>Добавить тариф</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="edit_tariff">
                <property name="text">
                 <string>Редактировать тариф</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delete_tariff">
                <property name="text">
                 <string>Удалить тариф</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="serv_b">
              <property name="text">
               <string>Показать все услуги</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;

</string>
              </property>
              <property name="text">
               <string>Информация о тарифе</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Название:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="tariff_name"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableWidget" name="services_t"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Стоимость абонемента:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="praise">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="sizeIncrement">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Длительность абонемента:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="duration">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="l_duration">
                  <property name="text">
                   <string>Возможность продления:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="addition"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="l_duration_praise">
                  <property name="text">
                   <string>Стоимость продления:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="addition_praise"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
